src/commands/functions/serve.js

```diff
@@ -0,0 +1,85 @@
+const { join } = require('path')
+
+const { flags: flagsLib } = require('@oclif/command')
+
+const Command = require('../../utils/command')
+const { getSiteInformation, acquirePort } = require('../../utils/dev')
+const { startFunctionsServer } = require('../../utils/serve-functions')
+
+const DEFAULT_PORT = 9999
+
+class FunctionsServeCommand extends Command {
+  async run() {
+    await this.config.runHook('analytics', {
+      eventName: 'command',
+      payload: {
+        command: 'functions:serve',
+      },
+    })
+
+    const { flags } = this.parse(FunctionsServeCommand)
+    const { error: errorExit, log, warn, netlify } = this
+    const { api, site, config, siteInfo } = netlify
+
+    const functionsDir =
```

Comment:
I think we're computing this directory in several places now. How do you feel about abstracting that into some sort of util method?

(Doesn't have to be part of this PR!)

Reply:
Good point! Done in https://github.com/netlify/cli/pull/2270/commits/878a9a5985bf3cff34f6c167812a4135b96f1c92

I also updated the `serve` command to use the same flag as the other ones (though I personally prefer `dir`), and updated the command descriptions to use `directory` instead of `folder` per our style guide.

I'll open a new issue to support `netlify/functions` for `list` and `build`.

Reply:
> I'll open a new issue to support `netlify/functions` for `list` and `build`.

See https://github.com/netlify/cli/issues/2319

Reply:
Great, thanks!